python/res/fm/ecl/ecl_run.py

```diff
@@ -255,7 +252,34 @@ def initMPI(self):
             for host in machine_list:
                 fileH.write("%s\n" % host)
 
-    def execEclipse(self):
+    def _get_run_command(self, eclrun_config):
+        return [
+            "eclrun",
+            "-v",
+            eclrun_config.version,
+            "--np",
+            str(self.num_cpu),
+            eclrun_config.simulator_name,
+            "{}.DATA".format(self.base_name)
+        ]
+
+    def _get_legacy_run_command(self):
```

Comment:
The challenge here is that all new versions has gotten a changed entry point with different way of running the executable. However this only applies for one of the simulators, and e.g. flow will be started through the "old" method still. So it isn't really a `legacy` run command.

Reply:
You are correct, both flow and eclipse versions older than `2018.2` will be run using the old method. So yes for flow it's not a "legacy" run command, but for eclipse it is :man_shrugging: Overall I don't like the fact that we are combining everything in how we run flow and eclipse, when they now differ in the way they are executed.

Reply:
Agree, flow should probably have gotten it's own way of execution.